assets/multiqc_config.yml

```diff
@@ -0,0 +1,14 @@
+report_comment: >
+  This report has been generated by the <a href="https://github.com/nf-core/magmap/tree/dev" target="_blank">nf-core/magmap</a> analysis pipeline. For information about how to
+  interpret these results, please see the <a href="https://nf-co.re/magmap/dev/docs/output" target="_blank">documentation</a>.
+report_section_order:
+  "nf-core-magmap-methods-description":
+    order: -1000
+  software_versions:
+    order: -1001
+  "nf-core-magmap-summary":
+    order: -1002
+
+export_plots: true
```

Comment:
Are you not using any MultiQC modules in for the pipeline?

Reply:
I don't know what you mean with this comment. We are using the nf-core module for MultiQC and we are just using it as it comes out from the template.